PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/IntelliJFunctionContext.java

```diff
@@ -175,6 +175,10 @@ public void setAppServicePlanName(String appServicePlanName) {
         this.appServicePlanName = appServicePlanName;
     }
 
+    public void setAppServicePlanResourceGroup(String appServicePlanResourceGroup) {
+        this.appServicePlanName = appServicePlanResourceGroup;
```

Comment:
appServicePlanName ??

Reply:
Yes, it should be appServicePlanResourceGroup, thanks~